integrations/extensions/starter-kits/servicenow/readme.md

```diff
@@ -0,0 +1,61 @@
+# [ServiceNow Incident Report]
+
+## Background
+
+
+ServiceNow is a cloud-based software platform for IT Service Management (ITSM) which helps to automate IT Business Management.
+It has a variety of functionalities, however this extension will focus on Incident management, specifically creating an Incident.
+
+Our focus would be on [ServiceNow Table API](https://developer.servicenow.com/dev.do#!/reference/api/sandiego/rest/c_TableAPI) and specifically on using `POST /now/table/{tableName}` where `tableName` is `incident`.
+
+This Extension is also described in the following [blog](update link).
+
+## Pre-Requisite Steps
+
+ServiceNow offers a Personal Developer Instance. See details [here](https://developer.servicenow.com/dev.do#!/learn/learning-plans/rome/new_to_servicenow/app_store_learnv2_buildmyfirstapp_rome_personal_developer_instances).
+### Auth Info
+The API requires authentication. To obtain the credentials login to your Developer Instance, and follow the following steps:
+
+1. At the top menu bar click on `All`
+2. On the filter, type in `User` and go to `System Security`->`Users and Groups`. 
+3. Search for `admin` under `UserID` 
+It should look similar to ![this](../servicenow/assets/ConfigureUser.png)
+Now,
+ - Change the email to your personal email 
+ - Check the `Passwords needs reset` box
+ - Click on `Update`
+ - Logout
+ - Try to Login, you should be offered to provide a new password, remember it as it wil be used alongside your username (`admin`) as means to do Basic Authentication.
```

Comment:
Tabbing, typos, and nitpicking

```suggestion
Click on the `admin` search result, then:
   - Change the email to your personal email 
   - Check the `Password needs reset` box
   - Click on `Update`
   - Logout
   - Try to log in with username `admin`. You should see an error banner and be offered reset your password through a link. Set a new password and remember it, as it will be used alongside your username (`admin`) for your Basic Authentication security configuration.
```